components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -36,6 +35,18 @@
   return [[UIColor blackColor] colorWithAlphaComponent:0.2f];
 }
 
+/**
+ The drawer height factor defines how much percentage of the screen space the drawer will take up
+ when displayed. The expected range is 0 - 1 (0% - 100%).
+
+ Default value is 0.5. If VoiceOver is enabled, the default value becomes 1.0.
+ */
+static CGFloat InitialDrawerHeightFactor(void) {
+  BOOL enableAccessibilityMode =
+      UIAccessibilityIsVoiceOverRunning() || UIAccessibilityIsSwitchControlRunning();
+  return enableAccessibilityMode ? 1.0f : 0.5f;
```

Comment:
Tip: Long-term I expect we may want to change this to an absolute unit of points for ergonomic reasons - e.g. a drawer should default to "x" distance where x has been researched as an optimal distance for a person's thumb to reach.

Reply:
yup makes sense. Currently we are following the design guidelines as seen here:
https://material.io/design/components/navigation-drawer.html#bottom-drawer

"Opening on mobile (portrait orientation)
As with other modal bottom sheets, the initial vertical position of a bottom navigation drawer is based on its content and screen height. They initially cannot open above 50% of the screen’s height.

If drawer contents are under 50% of screen height, open the drawer to full height at all times
If drawer contents are greater than 50% of screen height, open them to 50% initially, then allow a user to drag the drawer upward to its full height or screen height (whichever comes first)"

The percentage advantage is that the behavior is consistent across different device sizes.